src/components/Map/Map.js

```diff
@@ -216,16 +216,15 @@ class Map extends PureComponent {
   getQueryableLayers(featureInfoEventType) {
     const { layerService } = this.props;
 
-    return layerService
-      .getLayersAsFlatArray()
-      .filter(
-        (layer) =>
-          layer.getVisible() &&
-          layer.isQueryable &&
-          (
-            layer.get('featureInfoEventTypes') || ['pointermove', 'singleclick']
-          ).includes(featureInfoEventType),
+    return layerService.getLayersAsFlatArray().filter((layer) => {
+      return (
+        layer.visible &&
+        layer.isQueryable &&
+        (layer.get('f') || ['pointermove', 'singleclick']).includes(
```

Comment:
Wondering why `layer.get('featureInfoEventTypes')` has changed to `layer.get('f')`?

Reply:
Good catch! Sorry surely a Ctrl + f mistake :cold_sweat:

Reply:
Corrected :)